<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="18" ignored="0" total="33" passed="15">
  <reporter-output>
  </reporter-output>
  <suite name="MFAC_Suit" duration-ms="37287" started-at="2019-01-04T19:43:16Z" finished-at="2019-01-04T19:43:54Z">
    <groups>
    </groups>
    <test name="MFAC_Test" duration-ms="37287" started-at="2019-01-04T19:43:16Z" finished-at="2019-01-04T19:43:54Z">
      <class name="MFAC.MFAC">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:MFAC.MFAC@5f71c76a]" name="beforeClass" is-config="true" duration-ms="1" started-at="2019-01-04T19:43:16Z" finished-at="2019-01-04T19:43:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="VerifyPinValid(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinValid" duration-ms="27909" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinValid -->
        <test-method status="FAIL" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="1447" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:19Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"a8d881ff-b180-45f7-8e08-9a060d967cfc","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"a8d881ff-b180-45f7-8e08-9a060d967cfc","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="FAIL" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="2552" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"66db0677-6947-4613-bd2a-6fef7e185742","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"66db0677-6947-4613-bd2a-6fef7e185742","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="FAIL" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="3788" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"e0d446a5-f243-4ce7-847f-110a1a84eb91","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"e0d446a5-f243-4ce7-847f-110a1a84eb91","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="FAIL" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="4904" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"f7e4e7bb-c6bf-44ce-ada7-9e36eb623490","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"f7e4e7bb-c6bf-44ce-ada7-9e36eb623490","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="FAIL" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="6138" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"87937ed3-9969-4a99-873c-bf68109f36a2","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"87937ed3-9969-4a99-873c-bf68109f36a2","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="FAIL" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="7725" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"bf38a1a2-9d03-4f18-89d1-c5d07a931868","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"bf38a1a2-9d03-4f18-89d1-c5d07a931868","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="PASS" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="30995" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="PASS" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="31266" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="PASS" signature="IssuePin(java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="IssuePin" duration-ms="13958" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IssuePin -->
        <test-method status="FAIL" signature="IssuePin(java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="IssuePin" duration-ms="10088" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"6c69b9b2-f247-41e0-9f73-10b9f845feec","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"6c69b9b2-f247-41e0-9f73-10b9f845feec","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.IssuePin(MFAC.java:147)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IssuePin -->
        <test-method status="PASS" signature="IssuePin(java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="IssuePin" duration-ms="23885" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IssuePin -->
        <test-method status="FAIL" signature="AddressVelocity(java.lang.String, java.lang.String, java.lang.String, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="AddressVelocity" duration-ms="35747" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/addresses/velocitycheck]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"835fc60d-3185-4ad7-a1a2-db8ff9874962","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"835fc60d-3185-4ad7-a1a2-db8ff9874962","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.AddressVelocity(MFAC.java:128)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddressVelocity -->
        <test-method status="PASS" signature="VerifyPinNoLongerValid(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinNoLongerValid" duration-ms="35587" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinNoLongerValid -->
        <test-method status="FAIL" signature="VerifyPinValid(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinValid" duration-ms="21998" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"30d56f57-e408-4956-b31c-08dcac7b3006","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"30d56f57-e408-4956-b31c-08dcac7b3006","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinValid(MFAC.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinValid -->
        <test-method status="PASS" signature="IssuePinVelocity(java.lang.String, java.lang.String, java.lang.String, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="IssuePinVelocity" duration-ms="26533" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IssuePinVelocity -->
        <test-method status="FAIL" signature="VerifyPinNoLongerValid(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinNoLongerValid" duration-ms="11588" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinOTP"
     but: was "{"transactionId":"f6ee3490-5518-4728-beea-225a91b07660","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinOTP"
     but: was "{"transactionId":"f6ee3490-5518-4728-beea-225a91b07660","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinNoLongerValid(MFAC.java:229)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinNoLongerValid -->
        <test-method status="FAIL" signature="AddressVelocity(java.lang.String, java.lang.String, java.lang.String, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="AddressVelocity" duration-ms="35347" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/addresses/velocitycheck]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"193faa29-2535-489a-8fe7-ddd05088900d","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"193faa29-2535-489a-8fe7-ddd05088900d","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.AddressVelocity(MFAC.java:128)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddressVelocity -->
        <test-method status="FAIL" signature="IssuePin(java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="IssuePin" duration-ms="12678" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"589d3ba9-5730-4772-8571-ce694011137f","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"589d3ba9-5730-4772-8571-ce694011137f","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.IssuePin(MFAC.java:147)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IssuePin -->
        <test-method status="PASS" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="35180" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="PASS" signature="VerifyPinValid(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinValid" duration-ms="27963" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinValid -->
        <test-method status="PASS" signature="VerifyPinNoLongerValid(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinNoLongerValid" duration-ms="34798" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinNoLongerValid -->
        <test-method status="FAIL" signature="AddressVelocity(java.lang.String, java.lang.String, java.lang.String, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="AddressVelocity" duration-ms="34936" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/addressVelocityCheck]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"5a19df54-d08b-4117-87bd-a44d4476fbd9","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"5a19df54-d08b-4117-87bd-a44d4476fbd9","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.AddressVelocity(MFAC.java:128)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddressVelocity -->
        <test-method status="FAIL" signature="VerifyPinNoLongerValid(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinNoLongerValid" duration-ms="17737" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinOTP"
     but: was "{"transactionId":"786d47e3-646a-4e89-9a38-02352dc1aeeb","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinOTP"
     but: was "{"transactionId":"786d47e3-646a-4e89-9a38-02352dc1aeeb","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinNoLongerValid(MFAC.java:229)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinNoLongerValid -->
        <test-method status="FAIL" signature="VerifyPinValid(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinValid" duration-ms="19465" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"a567db67-c812-4abc-add8-5aae177c9cb4","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"a567db67-c812-4abc-add8-5aae177c9cb4","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinValid(MFAC.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinValid -->
        <test-method status="FAIL" signature="AddressVelocity(java.lang.String, java.lang.String, java.lang.String, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="AddressVelocity" duration-ms="34642" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/addressVelocityCheck]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"6e02498e-e70f-45f5-a5bf-a5b3f30d634b","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"6e02498e-e70f-45f5-a5bf-a5b3f30d634b","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.AddressVelocity(MFAC.java:128)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddressVelocity -->
        <test-method status="PASS" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="34516" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="PASS" signature="IssuePinVelocity(java.lang.String, java.lang.String, java.lang.String, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="IssuePinVelocity" duration-ms="34371" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IssuePinVelocity -->
        <test-method status="FAIL" signature="IssuePinVelocity(java.lang.String, java.lang.String, java.lang.String, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="IssuePinVelocity" duration-ms="15560" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"a8f9f8c7-3a9c-4cbb-b9ed-bd61df20e9e6","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"a8f9f8c7-3a9c-4cbb-b9ed-bd61df20e9e6","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.IssuePinVelocity(MFAC.java:165)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IssuePinVelocity -->
        <test-method status="FAIL" signature="IssuePinVelocity(java.lang.String, java.lang.String, java.lang.String, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="IssuePinVelocity" duration-ms="24969" started-at="2019-01-04T19:43:18Z" data-provider="dp" finished-at="2019-01-04T19:43:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-PHONE-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"2b62bd12-7701-45ee-9bdb-8b2cd0eecac0","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"2b62bd12-7701-45ee-9bdb-8b2cd0eecac0","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.IssuePinVelocity(MFAC.java:165)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IssuePinVelocity -->
        <test-method status="PASS" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="32692" started-at="2019-01-04T19:43:19Z" data-provider="dp" finished-at="2019-01-04T19:43:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[https://apidrt.idev.fedex.com:8443/security/v1/pin/verify]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
        <test-method status="PASS" signature="VerifyPinVelocity(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int, int)[pri:2, instance:MFAC.MFAC@5f71c76a]" name="VerifyPinVelocity" duration-ms="33380" started-at="2019-01-04T19:43:20Z" data-provider="dp" finished-at="2019-01-04T19:43:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FDM-POSTCARD-PIN]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[l7xx1892f99a6f88470ba29abc141cd7bd8d]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/issuePIN]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[http://mfacdrt-cos-vip.test.cloud.fedex.com:9090/mfac/v3/verifyPIN]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPinVelocity -->
      </class> <!-- MFAC.MFAC -->
      <class name="SupportClasses.Set_Environment">
        <test-method status="PASS" signature="SetEvironmentLevel(java.lang.String)[pri:0, instance:SupportClasses.Set_Environment@27fe3806]" name="SetEvironmentLevel" duration-ms="13" started-at="2019-01-04T19:43:16Z" finished-at="2019-01-04T19:43:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetEvironmentLevel -->
      </class> <!-- SupportClasses.Set_Environment -->
    </test> <!-- MFAC_Test -->
  </suite> <!-- MFAC_Suit -->
</testng-results>
